app/models/concerns/client_file_base.rb

```
@@ -64,13 +64,23 @@ def self.available_tags
     def as_preview
       return client_file.download unless client_file.variable?
 
-      client_file.variant(resize_to_limit: [1920, 1080]).processed.download
+      begin
+        client_file.variant(resize_to_limit: [1920, 1080]).processed.download
+      rescue ActiveStorage::FileNotFoundError
```

is it right to hide these exceptions? It seems like something has gone wrong if this is occurring. Maybe send to sentry?

---

I've only ever seen this development (because we clear our minio storage accidentally.)  Given that this is just the client headshot, I think it's ok.